ash/Cargo.toml

```diff
@@ -39,3 +39,16 @@ no-dev-version = true
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
+
+[lints.rust]
+deprecated_in_future = "warn"
+rust_2018_idioms = "warn"
+trivial_casts = "warn"
+trivial_numeric_casts = "warn"
+unused_qualifications = "warn"
+
+[lints.clippy]
+use_self = "warn"
+too_many_arguments = "allow"
```

Comment:
Unfortunately it is not possible to override lints from the root.
So here the common lints are duplicated to be able to add the three "allows"